targets/install.xml

```diff
@@ -281,8 +281,15 @@
                 <!-- Site-level setup -->
                 <!-- @see https://docs.acquia.com/cloud-platform/performance/memcached/enable/ -->
                 <httpget url="https://docs.acquia.com/_downloads/cloud-memcache-d8+.php" dir="${build.dir}/${drupal.root}/sites/${drupal.site.dir}" filename="settings.acquia-memcache.php"/>
+                <!-- Installing Cloud API SDK v2. -->
+                <composer command="suggest" composer="${composer.composer}">
+                  <arg line="--working-dir ${build.thebuild.dir}" />
+                  <arg value="--list" />
+                  <arg line="| xargs printf -- '%s:^2.0' | xargs composer require --working-dir ${application.startdir}" />
+                </composer>
+                <!-- Copy .env.example file to store Acquia Cloud Credentials -->
+                <copy file="${build.thebuild.dir}/defaults/install/.env.example" todir="${application.startdir}" overwrite="true"/>
```

Comment:
I know the `.env.example` currently only has Acquia data in it, but I could see it being useful for non-acquia hosted projects too, what do you think?

Should we copy the file somewhere more globally and add the platform specific data to it here instead?